rest-server/src/config/express.js

```diff
@@ -54,7 +54,7 @@ app.use((err, req, res, next) => {
   logger.warn('%s', err.stack);
   res.status(err.status || 500).json({
     message: err.message,
-    error: config.env === 'development' ? err.stack : {},
+    error: config.env === 'development' ? err.stack : '',
```

Comment:
Why?

Reply:
"error" field should be string type according to readme, but the default error handler returns a dict instead. Inconsistent json filed type will result to deserialization failure.

Reply:
OK. Which part in PAI will deserialize this field? If the existing implementation has not been introducing obvious problem to the system, I suggest we leave it as it is, in case we introduce more problems.